requirements/extras/test_requirements.txt

```diff
@@ -12,7 +12,7 @@ awslogs==0.14.0
 black==22.3.0
 stopit==1.1.2
 # Update tox.ini to have correct version of airflow constraints file
-apache-airflow==2.5.1
+apache-airflow<=2.5.2
```

Comment:
This should be `apache-airflow==2.5.2`

Reply:
If the whole point of doing this change is to upgrade the version to 2.5.2 then it is not making sense.

Reply:
+1 on Kalyani's comment, we'l need to resolve conflicts with patched version `apache-airflow==2.5.2`